pymc4/distributions/discrete.py

```diff
@@ -192,14 +195,15 @@ def _init_distribution(conditions):
             conditions["concentration1"],
         )
         return tfd.DirichletMultinomial(
-            total_count=total_count, concentration=[concentration0, concentration1]
+            total_count=total_count,
+            concentration=np.concatenate((concentration0, concentration1), axis=-1),
```

Comment:
Should we keep ``validate_args=True`` as the shapes are a little tricky to deal with?

Reply:
I'm not sure about it yet